packages/venia-concept/src/index.js

```diff
@@ -37,7 +38,10 @@ ReactDOM.render(
         store={store}
     >
         <WindowSizeContextProvider>
-            <App />
+            <ToastContextProvider>
+                <ToastContainer />
```

Comment:
As mentioned before, I would really like this to be a child of `App`, rather than a sibling. Is that doable?